packages/slidev/node/plugins/loaders.ts

```diff
@@ -154,12 +154,12 @@ export function createSlidesLoader(
         const moduleIds = new Set<string>()
 
         if (data.slides.length !== newData.slides.length) {
-          moduleIds.add('/@slidev/routes')
+          moduleIds.add('/@slidev/slides')
```

Comment:
I searched a bit and seems there are quite many users are using it, I prefer to not rename it if there is no hard reasons

Reply:
Resoved via a1f67110a882457b9bf6d6b1fb10730f9b32f731.

Also, if a `routeAlias` is defined, the URL of the slide will always be the `routeAlias`. Previously, `routeAlias` is only a redirect.